tljh/installer.py

```diff
@@ -99,15 +99,25 @@ def ensure_node():
     apt.add_source('nodesource', 'https://deb.nodesource.com/node_10.x', 'main')
     apt.install_packages(['nodejs'])
 
-
-def ensure_chp_package(prefix):
+def remove_chp():
     """
-    Ensure CHP is installed
+    Ensure CHP is not running
     """
-    if not os.path.exists(os.path.join(prefix, 'node_modules', '.bin', 'configurable-http-proxy')):
-        subprocess.check_output([
-            'npm', 'install', 'configurable-http-proxy@3.1.0'
-        ], cwd=prefix, stderr=subprocess.STDOUT)
+    if os.path.exists(os.path.join(HERE, 'systemd-units', 'configurable-http-proxy.service')):
```

Comment:
This is not the path where the unit is installed, which is what we want to check:

```suggestion
    if os.path.exists("/etc/systemd/system/configurable-http-proxy.service"):
```